eve/workers/openstack-multiple-nodes/terraform/nodes.tf

```diff
@@ -1,18 +1,9 @@
-resource "openstack_compute_servergroup_v2" "all" {
```

Comment:
Commit message could mention **WHY** we remove them.

And as they seems useless (as we just remove them and be done with it), maybe explain what was their role/why it's not an issue to simply remove them.

Reply:
Yep, good point, I wanted to see first if it does not break anything in the CI, but it looks good.